programs/vote/src/vote_state/mod.rs

```diff
@@ -1770,6 +1776,76 @@ mod tests {
         );
     }
 
+    #[allow(clippy::integer_arithmetic)]
+    fn process_new_vote_state_replaced_root_vote_credits(
+        feature_set: &FeatureSet,
+        expected_credits: u64,
+    ) {
+        let mut vote_state1 = VoteState::default();
+        for i in 0..MAX_LOCKOUT_HISTORY + 2 {
+            process_slot_vote_unchecked(&mut vote_state1, i as u64);
```

Comment:
nit: `process_slot_vote_unchecked` is outdated and uses `Vote`. Lets use `VoteStateUpdate` from now on, at least for newly written tests.

Reply:
Will do